graphics/libquirc/Portfile

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        dlbeer quirc 1.1 v
+name                lib${github.project}
+revision            0
+categories          graphics math
+license             ISC
+maintainers         nomaintainer
+supported_archs     arm64 x86_64
+
+description         QR decoder library
+long_description    QR codes are a type of high-density matrix \
+                    barcodes, and quirc is a library for extracting \
+                    and decoding them from images.
+
+homepage            https://github.com/dlbeer/quirc
+
+checksums           rmd160  45edf684de85d504631d3ea5b315b07ee8919895 \
+                    sha256  33cf10f6f567f07583014f2c8596915f890a78cd2d40f7d4ecaa380fa0d12f4a \
+                    size    38608
+
+depends_build-append \
+                    port:gmake
+
+# diff -NaurdwB ./quirc-orig ./quirc-new | sed -E -e 's/\.\/.+-(orig|new)\//\.\//' | sed -E -e 's|/opt/local|@PREFIX@|g' > ~/Downloads/patch-quirc.diff
+patchfiles          patch-quirc.diff
+
+post-patch {
+    reinplace "s|@MACPORTSUSER@|${macportsuser}|g" \
+                    ${worksrcpath}/Makefile
+    reinplace "s|@PREFIX@|${prefix}|g" \
+                    ${worksrcpath}/Makefile
+}
+
+variant quirc description {Full quirc installation.} {
+    depends_build-append \
+                    port:gmake
+
+    depends_lib-append  \
+                    port:libsdl \
+                    port:libsdl_gfx
+
+    post-patch {
+        xinstall -d ${workpath}/include
+
+        # https://raw.githubusercontent.com/openbsd/src/master/sys/sys/videoio.h
+        xinstall -m 0644 ${filespath}/videoio.h ${workpath}/include
+    }
+}
+
+compiler.cpath-append \
+                    ${workpath}/include
+configure.cflags-append \
+                    -I${prefix}/include/SDL \
+                    -I${workpath}/include
+
+use_configure       no
+
+build.cmd           gmake
+
+if { [variant_isset "quirc"] } {
+    build.args-append   BUILD_QUIRC=ON
+
+    destroot.post_args-append \
+                    PREFIX=${prefix}
+} else {
+    build.target    ${name}.dylib
+
+    destroot {
+        xinstall -m 0644 \
+                    ${worksrcpath}/${build_dylib} \
+                    ${destroot}${prefix}/lib/${build_dylib}
+        xinstall -m 0644 \
+                    ${worksrcpath}/lib/${github.project}.h \
+                    ${destroot}${prefix}/include
+    }
+}        
```

Comment:
I believe this line has trailing whitespace, which you'll want to fix.

And be sure to run `port lint --nitpick` one final time, to ensure there's nothing else.

Reply:
Thanks, all done.